ddtrace/appsec/_iast/_taint_tracking/TaintTracking/TaintedObject.cpp

```diff
@@ -30,14 +30,20 @@ shift_taint_range(const TaintRangePtr& source_taint_range, long offset)
 }
 
 void
-TaintedObject::add_ranges_shifted(TaintedObjectPtr tainted_object, long offset, int max_length, int orig_offset)
+TaintedObject::add_ranges_shifted(TaintedObjectPtr tainted_object,
+                                  RANGE_LENGTH offset,
```

Comment:
Just above in line 24 we use `offset` as `RANGE_START` instead of `RANGE_LENGTH`, it feels a bit confusing

Reply:
yes, it's confusing, that's why I'm adding RANGE_START and RANGE_LENGTH xD